portfolio/src/main/webapp/script.js

```diff
@@ -13,17 +13,24 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random facts to the page.
  */
-function addRandomGreeting() {
-  const greetings =
+function addRandomFacts() {
+  const facts =
       ['\'Wait for it...legendary\'', '\'Talent wins games, but teamwork and intelligence wins championships\'', 
       'Fact: I love sports, I practiced Taekwondo for 10 years', '\'Suit up\'', 'Fact: I have 3 dogs'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const random = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factsContainer = document.getElementById('container');
+  factsContainer.innerText = random;
 }
+
+/** Request content from server and add it to page */
+async function getUsingAsyncAwait() {
+  const response = await fetch('/data');
+  const data = await response.text();
+  document.getElementById('fetch-container').innerText = data;
+}
```

Comment:
nit: Please always add a new line at the end of the file

Reply:
Noted, thanks!